packages/transaction-controller/src/types.ts

```diff
@@ -44,10 +44,13 @@ type TransactionMetaBase = {
    */
   dappSuggestedGasFees?: DappSuggestedGasFees;
 
-  /**
-   * The default estimate for gas.
-   */
-  defaultGasEstimates?: string;
+  defaultGasEstimates?: {
```

Comment:
Is it beneficial to introduce a distinct type definition for `defaultGasEstimates`?

Reply:
Done.